src/client/components/AddReview.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import postData from "./postData";
+
+const AddReview = () => {
+    const [meal, setMeal] = useState([])
+    const [title, setTitle] = useState("");
+    const [description, setDescription] = useState("");
+    const [stars, setStars] = useState("");
+
+    const params = useParams();
+
+
+    useEffect(() => {
```

Comment:
Try and see if you can challenge yorself to create a global fetch method/Hook and store the url paths in an object.
You also need error handling and some sort of loading animation while fetching. This should be reusable also